### PR TITLE
Flow rest server

### PR DESCRIPTION
**Pull request by monica-adusumilli**
Rest server for flow.

### CODE REVIEW

File: flow_app.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Server for foundation app.
+from flask import Flask, escape, request
+import json
+import os
+from werkzeug.exceptions import NotFound
+import logging
+import requests
+
+FLOW_POLICY_FILE = "policy.json"
```
Comment (shivam-gupta-ntnx) [inline flow_app.py]: `config/policy.json`

File: flow_app.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+def home():
+  return "Stay Tuned for policy recommendations!", 200
+
+@app.route('/policy', methods = ['POST'])
+def fvm_post():
+  data = request.data
+  response = None
+  new_policy = json.loads(data)
+  if not new_policy:
+    raise NotFound("Please provide an FVM_IP")
```
Comment (shivam-gupta-ntnx) [inline flow_app.py]: FVM IP??

File: flow_app.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  if os.path.exists(FLOW_POLICY_FILE):
+    policy_list = json.load(open(FLOW_POLICY_FILE))
+  policy_list.append(new_policy)
+  with open(FLOW_POLICY_FILE, "w") as outfile:
+    outfile.write(json.dumps(policy_list, indent=2))
+  logger.info("Policies are %s")
+  response = "Successfully registered new policy %s" % new_policy
+  return json.dumps(response), 200
+
+@app.route('/policy', methods = ["GET"])
```
Comment (shivam-gupta-ntnx) [inline flow_app.py]: Policy get by ID

File: flow_app.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+def policy_get():
+  if os.path.exists(FLOW_POLICY_FILE):
+    response = json.load(open(FLOW_POLICY_FILE))
+  return json.dumps(response), 200
+
+@app.route('/recommendation', methods = ["GET"])
+def recommendations_get():
```
Comment (shivam-gupta-ntnx) [inline flow_app.py]: We should receive the vm list along with their types and tiers here, then only recommendation is possible.

Comment (monica-adusumilli) [reply flow_app.py]: We'll do that once recommendations and discovery is checked in

**Pull request merged**